setupTests.js

```diff
@@ -0,0 +1,4 @@
+var enzyme = require('enzyme');
```

Comment:
No reason to not have this as ES6. Rather use `const`. :)

Reply:
Done